evm_loader/tests/test_acc_storage_states.py

```diff
@@ -125,27 +125,29 @@ def neon_emv_instr_cancel_21(self, acc, caller, storage, nonce):
 
     def call_begin(self, storage, steps, msg, instruction, writable_code, acc, caller, add_meta=[]):
         print("Begin")
-        trx = Transaction()
-        trx.add(self.sol_instr_keccak(make_keccak_instruction_data(1, len(msg), 13)))
+        trx = TransactionWithComputeBudget()
+        self.index = len(trx.instructions)
```

Comment:
IMHO: it is better to rename `self.index` in something more clear. For example `self.first_instruction_index`.

Reply:
Renamed to `first_instruction_index` here and in other tests too.